docs/distribution/cli.rst

```diff
@@ -0,0 +1,8 @@
+.. _distribution-pipeline-cli:
```

Comment:
Minor organizational comment: Since the CLI is for both PDF and Distribution tasks, is there a better place to have this file live in terms of TOC so users don't think it only applies to the latter?